dash.js

```diff
@@ -165,17 +165,21 @@ const baseIconSizes = [16, 22, 24, 32, 48, 64, 96, 128];
  * - play animations even when not in overview mode
  * - set a maximum icon size
  * - show running and/or favorite applications
- * - emit a custom signal when an app icon is added
  * - hide showApps label when the custom menu is shown.
  * - add scrollview
  *   ensure actor is visible on keyfocus inseid the scrollview
  * - add 128px icon size, might be usefull for hidpi display
  * - sync minimization application target position.
  * - keep running apps ordered.
  */
-var MyDash = class DashToDock_MyDash {
```

Comment:
Yeah, not needed. Since we don't inherit from one of them, but still...

I also had some more changes I didn't push yet related to this, but I'll do in a follow-up MR